lib/CMakeCache.txt

```diff
@@ -0,0 +1,386 @@
+# This is the CMakeCache file.
```

Comment:
This file should also not be part of the git repository, it was created by cmake probably when you did an "in-source-build" which should be avoided!

Reply:
Deleted.